kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UIServicesClientImpl.java

```diff
@@ -130,12 +146,14 @@ private String getTaskFormByType( String containerId, Long taskId, String langua
             params.append( "&filter=" ).append( filter );
 
             return makeHttpGetRequestAndCreateRawResponse(
-                    build(loadBalancer.getUrl(), FORM_URI + "/" + TASK_FORM_GET_URI, valuesMap) + "?" + params.toString());
+                    build(loadBalancer.getUrl(), FORM_URI + "/" + TASK_FORM_GET_URI, valuesMap) + params.toString());
 
         } else {
-            CommandScript script = new CommandScript( Collections.singletonList(
-                    (KieServerCommand) new DescriptorCommand( "FormService", "getFormDisplayTask", new Object[]{containerId, taskId, StringUtils.defaultString( language ), !StringUtils.isEmpty( language ), formType } )) );
-            ServiceResponse<String> response = (ServiceResponse<String>) executeJmsCommand( script, DescriptorCommand.class.getName(), "BPM-UI", containerId ).getResponses().get(0);
+            CommandScript script = new CommandScript(Collections.singletonList(
+                    (KieServerCommand) new DescriptorCommand("FormService", "getFormDisplayTask",
+                            containerId, taskId, userId, StringUtils.defaultString(language), !StringUtils.isEmpty(language), formType)));
```

Comment:
@sutaakar  Sonar analysis advised to remove Object[]. I was making local testing but it went with this commit. Please advise if you see any problem with this change or if I should get back the Object[].

Reply:
Well, I would say you can keep there the Object[] to make implementation consistent with other methods and ignore this SonarCloud warning.

Reply:
Ok, will revert that change.